app/messaging/process-send-message.js

```diff
@@ -8,6 +9,7 @@ const processSendMessage = async (message, receiver) => {
     await receiver.completeMessage(message)
   } catch (err) {
     console.error('Unable to process message:', err)
+    await sendProcessFailureEvent(message.body.filename, err)
```

Comment:
This will also be called if the message fails validation.  We probably do want an error in that case too, so maybe if we change the template test to be more generic to the issue.

So instead of saying DAX cannot be contacted, say something like cannot publish file to DAX.  Then the error message in the body will explain the specific issue.

Reply:
Wording amended in the template:

![image](https://github.com/DEFRA/ffc-pay-file-publisher/assets/60177449/16da1522-9299-4b54-a0ab-f1212e757002)